cmd/cmd.go

```diff
@@ -98,8 +140,12 @@ func addDataDir(c *cobra.Command) *cobra.Command {
 }
 
 func addDatabase(c *cobra.Command) *cobra.Command {
-	c.Flags().StringVarP(&databaseURI, "database-uri", "u", "", "PostgreSQL URI (default DATABASE_URL environment variable)")
+
+	c.Flags().StringVarP(&databaseURI, "database-uri", "u", "", "Mongo URI (default MONGO_URL environment variable)")
+	c.Flags().StringVarP(&mongoDatabase, "database-name", "s", "minhareceita", "MongoDB Database")
+	c.Flags().StringVarP(&databaseURI, "database-uri", "u", "", "PostgreSQL URI (default POSTGRES_URL environment variable)")
```

Comment:
Não podemos ter duas _flags_ com o mesmo nome (e não precisamos do nome do banco de dados, como comentei anteriormente):

```suggestion
	c.Flags().StringVarP(&databaseURI, "database-uri", "u", "", "Database URI (default DATABASE_URL environment variable)")
```

Reply:
feito.